server/models/chat.js

```diff
@@ -0,0 +1,29 @@
+const db = require('../config/db');
+
+exports.create = ({ body, id_to, id_from }) => db.one(
+  `INSERT INTO message (body, sender_id, recipient_id)
+      VALUES ($1, $2, $3)
+      RETURNING message_id`,
+  [body, id_to, id_from]
+);
+
+exports.read = (account_id) => db.any(
```

Comment:
Maybe we should change this function to `list` instead of `read`, as we are now listing all conversations. It would also be good to have an additional `read` function that takes in a user id and gives all chats with that user.

Reply:
Would `listByRecipient` be a better name, or is that too verbose? I just think that it's not immediately clear what's being listed, but I'm open to other name suggestions.

Reply:
Yeah thats not a bad idea, I'm also open to just making a `chat.js` file after all and putting this as a `list` method in that file. Either way, your call, being too verbose is a good place to be compared to being too terse here imo.